src/booleans.js

```diff
@@ -1,62 +1,68 @@
 function negate(a) {
-  // your code here
+  return !a;
+  //
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return !a && !b;
 };
 
 function one(a, b) {
-  // your code here
+  return (a&&!b) || (b&&!a);
 };
 
 function truthiness(a) {
-  // your code here
+  return !!a;
 };
 
 function isEqual(a, b) {
-  // your code here
+  return !!(a==b) ;
```

Comment:
You could simplify this by just doing `a === b`. 
You should also always use `===` where possible over `==`. It's very rare you would use `==` in real life situations.